cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,22 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        while True:
+            try:
+                connection.ensure_connection()
+            except OperationalError:
+                self.stdout.write(
+                    "Failed to find database"
+                )
+                time.sleep(5)
+            else:
+                break
+
+        self.stdout.write(self.style.ERROR("Database found!"))
```

Comment:
The message 'Database found!' is marked as an error (`self.style.ERROR`). It seems to me that this should be a success message, not an error. Consider changing this to a success message.